packages/@storefront/breadcrumbs/test/unit/refinement-crumbs.ts

```diff
@@ -27,28 +28,78 @@ suite('RefinementCrumbs', ({ expect, spy, stub, itShouldProvideAlias }) => {
       expect(updateState).to.be.called;
     });
 
-    it('should set the navigationSelector and selectedRefinementsUpdated based on SEARCH storeSection', () => {
-      const select = (refinementCrumbs.select = spy());
-      refinementCrumbs.props.storeSection = StoreSections.SEARCH;
-      refinementCrumbs.updateState = () => null;
-      refinementCrumbs.init();
+    describe('SEARCH storeSection', () => {
+      beforeEach(() => {
+        refinementCrumbs.props.storeSection = StoreSections.SEARCH;
+        refinementCrumbs.updateState = () => null;
+      });
+
+      it('should set the navigationSelector and selectedRefinementsUpdated', () => {
+        const select = (refinementCrumbs.select = spy());
+        refinementCrumbs.init();
+
+        refinementCrumbs.state.navigationSelector(field);
+
+        expect(refinementCrumbs.state.selectedRefinementsUpdated).to.eq(Events.SELECTED_REFINEMENTS_UPDATED);
+        expect(select).to.be.calledWithExactly(Selectors.navigation, field);
+      });
+
+      it('should listen for NAVIGATIONS_UPDATED', () => {
+        const subscribe = refinementCrumbs.subscribe = spy();
+
+        refinementCrumbs.init();
+
+        expect(subscribe).to.be.calledWithExactly(Events.NAVIGATIONS_UPDATED, refinementCrumbs.updateRefinements);
+      });
 
-      refinementCrumbs.state.navigationSelector(field);
+      // tslint:disable-next-line max-line-length
+      it('should not listen for NAVIGATIONS_UPDATED if the selectedRefinements prop is available', () => {
+        const selectedNavigation: any = {};
+        const subscribe = refinementCrumbs.subscribe = spy();
+        refinementCrumbs.props = { ...refinementCrumbs.props, field, selectedNavigation };
 
-      expect(refinementCrumbs.state.selectedRefinementsUpdated).to.eq(Events.SELECTED_REFINEMENTS_UPDATED);
-      expect(select).to.be.calledWithExactly(Selectors.navigation, field);
+        refinementCrumbs.init();
+
+        expect(subscribe).to.not.be.called;
+      });
     });
 
-    it('should set the navigationSelector and selectedRefinementsUpdated base on PAST_PURCHASES storeSection', () => {
-      const select = (refinementCrumbs.select = spy());
-      refinementCrumbs.props.storeSection = StoreSections.PAST_PURCHASES;
-      refinementCrumbs.updateState = () => null;
-      refinementCrumbs.init();
+    describe('PAST_PURCHASES storeSection', () => {
+      beforeEach(() => {
+        refinementCrumbs.props.storeSection = StoreSections.PAST_PURCHASES;
+        refinementCrumbs.updateState = () => null;
+      });
+
+      it('should set the navigationSelector and selectedRefinementsUpdated', () => {
+        const select = (refinementCrumbs.select = spy());
```

Comment:
Will the linter complain if we remove these parentheses? I think these parentheses were a product of having prettier.

Reply:
Nope it did not complain. I will remove these. Resolving conversation.